README.md

```diff
@@ -95,11 +95,11 @@ inc_bench
 #### Intel® Neural Compressor supports CPUs based on [Intel 64 architecture or compatible processors](https://en.wikipedia.org/wiki/X86-64):
 
 * Intel Xeon Scalable processor (formerly Skylake, Cascade Lake, Cooper Lake, and Icelake)
-* Future Intel Xeon Scalable processor (code name Sapphire Rapids)
+* Intel® Max Series CPUs
 
 #### Intel® Neural Compressor supports GPUs built on Intel's Xe architecture:
 
-* [Intel® Data Center GPU Flex Series](https://www.intel.com/content/www/us/en/products/docs/discrete-gpus/data-center-gpu/flex-series/overview.html)
+* Intel® Max Series GPUs
```

Comment:
Intel Xeon CPU Max Series (formerly Sapphire Rapids HBM)
Intel Data Center GPU Flex Series (formerly Arctic Sound-M)
Intel Data Center GPU Max Series (formerly Ponte Vecchio)

Reply:
Intel Xeon Scalable processor (formerly Skylake, Cascade Lake, Cooper Lake, Ice Lake, and Sapphire Rapids)